action.yml

```
@@ -17,13 +17,30 @@ runs:
     # * https://github.com/JuliaLang/Pkg.jl/issues/2011
     # * https://github.com/JuliaRegistries/General/issues/16777
     # * https://github.com/JuliaPackaging/PkgServer.jl/issues/60
-  - run: julia --color=yes -e 'using Pkg; VERSION >= v"1.5-" && !isdir(joinpath(DEPOT_PATH[1], "registries", "General")) && Pkg.Registry.add("General")'
+  - run: |
+      julia --color=yes -e '
+        using Pkg
+        if VERSION >= v"1.5-" && !isdir(joinpath(DEPOT_PATH[1], "registries", "General"))
+          function f()
```

This won't work as expected. Function defs in `if` blocks are executed no matter what. Since there's no `else` block, this won't actually cause issues but I think it would be nicer to move the function out of the if block regardless, and give it a descriptive name.

```suggestion
        VERSION >= v"1.5-" || exit(0)
        
        if !isdir(joinpath(DEPOT_PATH[1], "registries", "General"))
          function f()
```

---

Done